### DISCUSSION TITLE
Send transactions and SmartContracts

### DISCUSSION THREAD

**Discussion by LuisRibeiroo** [WalletConnect/walletconnect-monorepo#4505] [Developer Support]
Hello, everyone. I'm pretty new on web3 and still trying to get some basic items.
I have few questions i couldn't found answer 'till now:

**1.** Can I send a transaction with specific token (like USDC or USDT) without using SmartContract?
**1.1.** If so, how could I pass the token trough the sendTransaction request?
**1.2.** If don't, there's any way to send specific tokens without SC?

**2.** How's the common/recommended way to get the ABI contracts for each token?

**Comment by quetool** [comment]
This discussion is taken place [here](https://github.com/WalletConnect/Web3ModalFlutter/issues/126#issuecomment-2102739359) already! Closing this for simplicity. Thanks!
